frontend/src/components/public_group/PublicGroupHero.js

```diff
@@ -1,7 +1,33 @@
 import React from 'react';
 import PublicTopBarV2 from '../../containers/PublicTopBarV2';
+import formatCurrency from '../../lib/format_currency';
 
 export default class PublicGroupHero extends React.Component {
+
+  renderHeroStatistics()
+  {
+    const { group } = this.props;
+    if (!group.backers.length) return;
+    const backerCount = group.backers.reduce((prev, curr) => prev + (curr.role === 'BACKER'), 0);
```

Comment:
complicated way to compute this.
why don't you use `filterCollection(group.backers, {tier: tier.name})` with `import filterCollection from '../../lib/filter_collection';`